.github/workflows/pr-test-lint.yml

```diff
@@ -81,6 +81,27 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
+      - name: Updates to cnquery core go.mod?
+        id: cnquerygomod
+        if: github.event_name == 'pull_request' && github.actor == 'dependabot[bot]'
+        run: |
+          echo "COUNT_GOMOD=$(git diff-tree --name-only -r HEAD~1..HEAD | grep -c -E "^go.mod$")" >> $GITHUB_OUTPUT
+
+      - name: Fix providers go.mod for dependabot PRs
```

Comment:
Just to fully understand this approach, this is pushing a commit to the dependabot PR that is open?

Reply:
Yes, that's what it should do from what I could read about it.

From my understanding, this is possible with newer versions of the `checkout` action. It checks out the branch of the current PR. In case it is a dependabot PR that changes `/go.mod`, this adds a commit that updates the providers `go.mod` files.

This should then allow the following tests to pass. We get the tests across the complete PR and later have one large PR, including all the updates.

Reply:
Lets try it.